### DISCUSSION TITLE
Scheduling refresh

### DISCUSSION THREAD

**Discussion by cohawkeye** [code4recovery/12-step-meeting-list#417] [Q&A] [answered]
Hello

I might be missing something, but I have to come in daily and refresh my data source.  Is there a way to schedule this or automate it?  My updates come from a separate JSON and then I have to update it daily.  
Thanks.

**Comment by joshreisner** [comment, marked as answer]
HI @cohawkeye are you talking about refreshing your JSON feed in WordPress on the Import & Settings page?

**Reply by cohawkeye** [reply]
I am.  Am I missing some way to automate that?

**Reply by joshreisner** [reply]
Nope! It would require a big code update, because WordPress is hard to automate for background tasks, especially ones that could time out like importing. That's why you have to keep on logging in and clicking refresh. 

There is a ticket about this, so you can follow updates there: https://github.com/code4recovery/12-step-meeting-list/issues/42

Also there is a pull request: https://github.com/code4recovery/12-step-meeting-list/pull/66

Hope you don't mind I'm going to move this discussion over to the 12-step-meeting-list (wordpress plugin) repository

**Reply by cohawkeye** [reply]
Awesome.  Thanks for the reply and thanks for your service!
